### ISSUE TITLE
Off And On Again - 01006F5013202000

### ISSUE THREAD

**Issue by EmulationFanatic** [Ryujinx/Ryujinx-Games-List#4144]
#### Graphics Backend : Vulkan

Loads and plays at a high FPS.

#### Screenshots :
![2022-10-29 06_17_51-Ryujinx 1 1 331 - Off And On Again v1 4 (01006F5013202000) (64-bit)](https://user-images.githubusercontent.com/62343878/198831261-88cf54d9-6f94-4a79-9803-4039635e0214.png)
![2022-10-29 06_18_19-Ryujinx 1 1 331 - Off And On Again v1 4 (01006F5013202000) (64-bit)](https://user-images.githubusercontent.com/62343878/198831263-4950260b-7a46-4f5b-8f17-270f3afbcb42.png)
![2022-10-29 06_18_52-Ryujinx 1 1 331 - Off And On Again v1 4 (01006F5013202000) (64-bit)](https://user-images.githubusercontent.com/62343878/198831266-fe2c7d9f-4c98-45ba-a999-d0e3e42a7eca.png)
![2022-10-29 06_19_47-Ryujinx 1 1 331 - Off And On Again v1 4 (01006F5013202000) (64-bit)](https://user-images.githubusercontent.com/62343878/198831271-17aad8cf-f797-411f-bbf4-fa835cf5c1a5.png)
![2022-10-29 06_20_37-Ryujinx 1 1 331 - Off And On Again v1 4 (01006F5013202000) (64-bit)](https://user-images.githubusercontent.com/62343878/198831276-8a4924e4-da8f-468e-b49c-023ef51c1022.png)
![2022-10-29 06_21_01-Ryujinx 1 1 331 - Off And On Again v1 4 (01006F5013202000) (64-bit)](https://user-images.githubusercontent.com/62343878/198831277-b8cde1d3-059b-4b44-9857-3ccc2a626ab0.png)
![2022-10-29 06_21_30-Ryujinx 1 1 331 - Off And On Again v1 4 (01006F5013202000) (64-bit)](https://user-images.githubusercontent.com/62343878/198831283-a9ba4edb-0501-44dc-b8df-defeb9c3d195.png)

**Comment by grenmonika923** [comment]
I am a gambler and I am looking for a reliable resource with interesting and safe games. I love drive and fun. I hope to find like-minded people and chat on a reliable gaming platform.

**Comment by Gabriella971** [comment]
Of course, in the rapidly developing world of games, it is very easy to get lost. But still, in my opinion, games and the lottery add a little excitement and pleasure to our lives. This is a very interesting topic, thanks to [powerball](https://power486.com/) I was able to earn my first money. I still remember that feeling of euphoria and celebration that has been with me for a long time. Most people tend to go to community sites to find information, and this is the right decision. Here you can always find like-minded people with whom it is always interesting and fun.
